common/persistence/visibility/elasticsearch/processor.go

```diff
@@ -35,14 +35,14 @@ import (
 
 	"github.com/dgryski/go-farm"
 	"github.com/olivere/elastic/v7"
+	client2 "go.temporal.io/server/common/persistence/visibility/elasticsearch/client"
```

Comment:
client2?

Reply:
This is GoLand. I am fixing all of these.